nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -75,6 +75,24 @@ services.xserver.displayManager.defaultSession = "xfce+icewm";
 </programlisting>
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The testing driver implementation in NixOS is now in Python <filename>make-test-python.nix</filename>.
+     This was done by Jacek Galowicz <link xlink:href="https://github.com/tfc">(@tfc)</link>, and with the
+     collaboration of Julian Stecklina <link xlink:href="https://github.com/blitz">(@blitz)</link> and
+     Jana Traue <link xlink:href="https://github.com/jtraue">@jtraue</link>. All documentation has been updated to use this
+     testing driver, and a total of [how_many_tests_were_ported] were ported. In 20.09 the Perl driver implementation,
```

Comment:
Did not we want one more buffer release?

Reply:
I believe it was agreed widely that 20.09 was when it was going to be removed.
If anyone really needs it after that, or they having interest in the driver, they need to communicate this. The community has no incentive to keep it past 6 more months.